app/api/dao/user.py

```diff
@@ -143,27 +147,30 @@ def get_user_by_username(username: str):
         return UserModel.find_by_username(username)
 
     @staticmethod
-    def list_users(user_id: int, search_query: str = "", is_verified = None):
+    def list_users(user_id: int, search_query: str = "", page: int = DEFAULT_PAGE, per_page: int = DEFAULT_USERS_PER_PAGE, is_verified = None):
         """ Retrieves a list of verified users with the specified ID.
         
         Arguments:
             user_id: The ID of the user to be listed.
             search_query: The search query for name of the users to be found.
             is_verified: Status of the user's verification; None when provided as an argument.
+            page: The page of users to be returned
+            per_page: The number of users to return per page
         
         Returns:
             A list of users matching conditions and the HTTP response code.
         
         """
 
-        users_list = UserModel.query.filter(UserModel.id != user_id).all()
+        users_list = UserModel.query.filter(
```

Comment:
@lmcicat I am thinking, the return of the users is not apparently ordered, could you order it by id? In that way I can validate if the logic of the users returned by page, is correct or not.

By the way, thank you so much for your patience here @lmcicat 🙏This is a very important feature, that can help majorly this API to scale.

Reply:
No problem at all, I am happy to do it! Just updated the code to order by id.